Test/Nat/Max.expected

```diff
@@ -22,6 +22,7 @@ query:
 (declare-const y Nat)
 (assert (>= y 0))
 (define-fun Max ((x Nat) (y Nat)) Nat (ite (<= x y) y x))
+(assert (forall ((_uniq.231 Nat)) (=> (>= _uniq.231 0) (forall ((_uniq.232 Nat)) (=> (>= _uniq.232 0) (>= (Max _uniq.231 _uniq.232) 0))))))
```

Comment:
Output change 2: we now encode the inequality conditions for `define-fun`s as well as `declare-fun`. This seems correct to me, but maybe it is a regression?